index.js

```diff
@@ -9,7 +6,14 @@ module.exports = robot => {
     const issueOwner = issue.user.login;
     const repo = context.repo({username: issueOwner});
 
-    const config = await configs(context, './lib/defaults');
+    let config;
+
+    try {
+      config = await context.config('teacherbot.yml');
+    }
+      catch(err) {
+        config = Object.assign( {}, require('./lib/defaults.js') || {} );
```

Comment:
I would keep the defaults in a separate variable but in the same file. Keeps the fucntionality separate from static stuff like text.